module/os/freebsd/zfs/zfs_vnops.c

```diff
@@ -36,6 +36,7 @@
 #include <sys/sysmacros.h>
 #include <sys/resource.h>
 #include <sys/vfs.h>
+#include <sys/endian.h>
```

Comment:
Did you mean to include this in the PR?

Reply:
It was implicitly included before. Won't build without.